editor.planx.uk/src/components/AnalyticsDisabledBanner.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+
+import AnalyticsDisabledBanner from "./AnalyticsDisabledBanner";
+
+const meta = {
+  title: "Design System/Molecules/AnalyticsDisabledBanner",
+  component: AnalyticsDisabledBanner,
+} satisfies Meta<typeof AnalyticsDisabledBanner>;
+
+type Story = StoryObj<typeof meta>;
+
+export default meta;
+
+/** Add `&analytics=false` to the end this page's URL to display the banner */
```

Comment:
JSDoc comments translate to Storybook 7 "descriptions" :sparkles: see here for more https://storybook.js.org/docs/react/api/doc-block-description#writing-descriptions

Reply:
This is a nice feature!